app/uk/gov/hmrc/helptosavetestadminfrontend/views/layout.scala.html

```diff
@@ -0,0 +1,50 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.hmrcfrontend.views.html.helpers.HmrcReportTechnicalIssueHelper
+@import uk.gov.hmrc.hmrcfrontend.views.html.helpers.HmrcLayout
+@import uk.gov.hmrc.hmrcfrontend.views.config.StandardBetaBanner
+@import uk.gov.hmrc.hmrcfrontend.views.html.helpers.HmrcAccessibleAutocompleteCss
+@import uk.gov.hmrc.hmrcfrontend.views.html.helpers.HmrcAccessibleAutocompleteJavascript
+@import uk.gov.hmrc.helptosavetestadminfrontend.config.AppConfig
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+
+@this(
+        hmrcLayout: HmrcLayout,
+        standardBetaBanner: StandardBetaBanner,
+        appConfig: AppConfig,
+        hmrcReportTechnicalIssueHelper: HmrcReportTechnicalIssueHelper,
+        autcompleteCss: HmrcAccessibleAutocompleteCss,
+        autocompleteJavascript: HmrcAccessibleAutocompleteJavascript
+)
+
+@(title: String, pageScripts: Option[Html] = None, pageHeadBlock: Option[Html] = None)(contentBlock: Html)(implicit request : Request[_], messages: Messages, config: AppConfig)
+
```

Comment:
Can  'config: AppConfig' be removed please, doesn't seem it's used.

Reply:
Looks like the Curl Result Page needs to be fixed 
<img width="620" alt="Screenshot 2023-01-19 at 12 42 43" src="https://user-images.githubusercontent.com/86231805/213445800-9aedfcbb-d053-4d92-9b25-7e72e6719e7c.png">


Also the error messages do now show for CreateAccount , CheckEligibity and GetAccount Pages.

The error message for delete emails doesn't seem right.
<img width="522" alt="Screenshot 2023-01-19 at 12 44 52" src="https://user-images.githubusercontent.com/86231805/213446219-14f4298b-dd38-43c0-8daa-d06636d3d2fd.png">

Reply:
Yes on this service there is almost zero validation of form submissions hence no errors being returned - I think we need a discussion about what to do about this given it's an internal tool and not a public facing service

Reply:
I thought I might be going overboard introducing error summaries as there was only a collection of spans per error previously but then thought obvs can't leave like that even if internal as screenreaders will not be able to make sense of the page in the same way they can with error summaries

Reply:
I will fix up the curl result page to be more on pattern for sure though